sentences/nl/homeassistant_HassTurnOn.yaml

```diff
@@ -7,4 +7,8 @@ intents:
           - "schakel <name>[ ][<type>] [<naar>] in"
           - "[<zou>] <name>[ ][<type>] [willen|kunnen] (aan[ ]zetten|inschakelen)"
         excludes_context:
-          domain: cover
+          domain:
+            - cover
+            - scene
+            - sensor
+            - script
```

Comment:
Remove this. You wouldn't want to turn on all scripts tied to a room!

Reply:
Unless I misunderstood what `excludes_context` does, I would say that adding `script` and `scene` here excludes them from being targeted by this sentence. I would say they need to be in here, because `script` and `scene` now have their own files.
I have removed sensor, as they can't be turned on.